src/app/Fake.FluentMigrator/Sample/Sample.fsx

```diff
@@ -0,0 +1,30 @@
+﻿#r @"..\..\..\..\packages\FluentMigrator\lib\40\FluentMigrator.dll"
+#r @"..\..\..\..\packages\FluentMigrator.Runner\lib\40\FluentMigrator.Runner.dll"
+#r @"..\..\..\..\build\FakeLib.dll"
+
+#load "..\FluentMigratorHelper.fs"
```

Comment:
Same here. How can users find the sample?

Reply:
The sample is leftover from the migration, I probably should have removed this.

I will either fix and make sure the sample works and call it out in documentation, or remove it entirely.